cms/toolbar/toolbar.py

```diff
@@ -24,7 +23,7 @@
 
 
 class CMSToolbarLoginForm(AuthenticationForm):
-    username = forms.CharField(label=_("Username"), max_length=100)
+    username = forms.CharField(max_length=100)
```

Comment:
Legacy code from the days django had very short usernames. django CMS does not define a user, so we can now get rid of it and just use whatever Django provides